entry/src/main/js/default/pages/index/index.css

```diff
@@ -0,0 +1,77 @@
+#page {
```

Comment:
change the style names to .page. .card etc.,   to start with dot and test once in emulator.. if any unused ones can be removed.